### DISCUSSION TITLE
How can I set up a project with DCRF?

### DISCUSSION THREAD

**Discussion by devrrior** [NilCoalescing/djangochannelsrestframework#118] [Q&A] [answered]
Hi, I want to use DCRF for a personal project, but I don't know how can I start using this framework, for example, what I have to write in my settings.py and topics related. Mainly because there are no videos or posts that explain this framework. It would be great if you could provide me with some help.

**Comment by hishnash** [comment, marked as answer]
The best thing to do is to start out with the Channels tutorial https://channels.readthedocs.io/en/stable/tutorial/index.html for DCRF you do not need to set up anything in your settings files, just install the package and use it when building your consumers. See https://djangochannelsrestframework.readthedocs.io/en/v1.0.0/tutorial/part_1.html

**Comment by johnthagen** [comment]
@hishnash I think a simple full example project that uses DCRF would be a really helpful resource/ reference for users of this project.

Maybe if the tutorial was hosted on a GitHub repo?

**Reply by johnthagen** [reply]
One example I found: https://github.com/theY4Kman/dcrf-client/tree/master/test/integration/dcrf_client_test
